R/funding-element.R

```diff
@@ -37,7 +38,7 @@ add_funding <- function(parent_element, funder_name, funder_identifier, award_nu
     fund_error_message <- switch(fund_error, funder_name = "Please provide funders name.",
                                  funder_identifier = "Please provide funder identifier link.",
                                  award_number = "Please provide your award number.", 
-                                 award_title = "Please provide the title of your project.",
+                                 award_title = "Please provide the title of your award.",
```

Comment:
Are you sure that award is more appropriate than project here?

Reply:
Its the title of your project that received the award?

Reply:
The reason I changed it to award title instead of project title is because there is a required project title element within the project node. We end up having a project title and an award title that we have to include so I thought it was best to specify that within funding it is the award title instead of the project title.

Reply:
good call